### ISSUE TITLE
Controller with return method in superclass, Around interceptor called several times, ok with 1.2.6, ko with 1.3.3

### ISSUE THREAD

**Issue by jrevault** [micronaut-projects/micronaut-core#3041]
Hi, 
I have a strange behavior with an around interceptor when my endpoints is calling a superclass to return data to the client.
I highlighted that into a simple project :
[https://github.com/jrevault/micronaut_intercept_issue](https://github.com/jrevault/micronaut_intercept_issue)

Basically with Micronaut version 1.3.3 the around interceptor will be called several times (seems that it depends on the number of method that the return goes into), while with Micronaut 1.2.6 it's working perfectly.

Basically I have an Endpoint class extends EndpointSuperclass and has an around interceptor.
2 endpoints a & b.
**Endpoint a** returns directly to the client : works fine
**Endpoint b** returns through EndPoint superclass : around interceptor is called several times

Any thoughts ?
Thanks

**Comment by jameskleeh** [comment]
This is how it should be working. It's unfortunate that it changed this way from 1.2 to 1.3, however it has been some time since then so I don't think its reasonable to revert to the previous behavior. If you only want the `a` and `b` methods to be intercepted, apply the annotation to those methods only.
